search-services/latest/config/performance.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Performance Recommendations
+---
+
+From version 2.0.2, the custom `<SOLR6_INSTALL_LOCATION>/contentstore` folder has been removed. Metadata, Permissions, and Content for the Alfresco Repository nodes are fully stored in the SOLR Core standard indexes.
+
+Since the total amount of storage could be equivalent between previous versions, the SOLR Core Index storage has been increased. In order to control the size and the performance of the SOLR index, the following actions may be performed:
+
+* [Disable FINGERPRINT](#disable-fingerprint) reduces storage requirements
+* [Disable SOLR Document Cache](#disable-solr-document-cache) reduces RAM requirements
+* [Optimize SOLR Index](#optimize-solr-index) improves search performance
+
+Additionally, from version 2.0.2, SOLR Merging parameters have been exposed that can be configured within the `solrcore.properties` file. The default values function adequately for many use cases, however some recommendations are given to increase performance in large deployments.
+
+## Disable FINGERPRINT
+
+The [Document Fingerprints]({% link search-services/latest/admin/index.md %}#disable-fingerprint) feature can be used to get similar documents from SOLR using the reserved word `FINGERPRINT` in `FTS` search syntax. In order to provide these results, each document in the SOLR Index includes a list of `MINHASH` fields that create larger Lucene Indexes.
```

Comment:
Link (search-services/latest/admin/index.md %}#disable-fingerprint) seems to be incorrect?

Reply:
@eliaporciani can you confirm if this link is correct please?
I presume I got it from you?
Cheers :-)

Reply:
Done